composer.json

```diff
@@ -17,13 +17,17 @@
         {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
         {"name": "Johannes Schmitt", "email": "schmittjoh@gmail.com"}
     ],
+    "suggest": {
+        "vimeo/psalm": "Psalm allows you to enforce template types for your collections."
```

Comment:
This is to be dropped - Psalm is not a tool for production, just like PHPStan or Doctrine CS are not.
Feel free to open issue on Composer repo to add `suggest-dev` but it doesn't belong to `suggest`.

Reply:
Agreed: this will end up just being noise